addon/serializer.js

```diff
@@ -0,0 +1,3 @@
+import Serializer from "ember-data/-private/system/serializer";
+
+export default Serializer;
```

Comment:
@pangratz could you move the `Serializer` and `EmbeddedRecordsMixin` out of the `-private` folder and into their new public locations.